action.yml

```
@@ -24,5 +24,22 @@ inputs:
     default: main
 
 runs:
-  using: 'docker'
-  image: 'Dockerfile'
+  using: "composite"
+  steps:
+    - name: Set up pixi
+      uses: prefix-dev/setup-pixi@ba3bb36eb2066252b2363392b7739741bb777659  # v0.8.1
+      with:
+        cache: true
```

```suggestion
        locked: true
        cache: true
```

Explicitly enforce the default (of `locked` being used) in the event that somehow changes in the future.

c.f. https://github.com/scientific-python/upload-nightly-action/pull/91/files#r1777356588